hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieMultiTableDeltaStreamer.java

```diff
@@ -258,7 +258,6 @@ public static class Config implements Serializable {
         description = "base path prefix for multi table support via HoodieMultiTableDeltaStreamer class")
     public String basePathPrefix;
 
-    @Parameter(names = {"--target-table"}, description = "name of the target table", required = true)
```

Comment:
Do you check if this change would bring the compatibility problem?

Reply:
There will be compatibility problems with the old version. This parameter is required in the old version. If this parameter is removed but configured in the command, an error will be reported. Do we have to be compatible with older versions?

Reply:
@yanghua I think it's a good idea. Thank you for your suggestion. I have resubmitted the new code

Reply:
thanks, LGTM.